auslib/db.py

```diff
@@ -1909,13 +1912,18 @@ def getBlob():
             except IndexError:
                 raise KeyError("Couldn't find release with name '%s'" % name)
 
+        def get_data_version(obj):
```

Comment:
Was necessary because py3 does not compare dicts. See bellow:

```
ex-dev@exdev-moz:~/code.exdev/mozilla/releng/balrog$ python2
Python 2.7.12 (default, Dec  4 2017, 14:50:18) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> a = {'data_version': 1}
>>> b = {'data_version': 1}
>>> a > b
False
>>> 
ex-dev@exdev-moz:~/code.exdev/mozilla/releng/balrog$ python3.6
Python 3.6.5 (default, Mar 29 2018, 03:28:50) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = {'data_version': 1}
>>> b = {'data_version': 1}
>>> a > b
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'dict' and 'dict'

```